tests/installation/ipxe_install.pm

```diff
@@ -307,6 +336,10 @@ sub run {
         else {
             record_info("Installing", "Pls check if the expected product is being installed.");
         }
+        if (is_agama) {
+            assert_screen('agama-installer-live-root', 400);
+            return;
+        }
```

Comment:
I have removed this `if() else` block in my another PR. You need not add this `if` statement at here either.